### DISCUSSION TITLE
Can you extract raw binary audio data from AudioContext?

### DISCUSSION THREAD

**Discussion by dmdum** [WebAudio/web-audio-api#2521] [Q&A] [answered]
I'm still in the investigation phase of this but basically, I want to extract the binary audio data from a conference call. We're using 100ms for conference call and was referred to their [Custom Audio Plugin](https://www.100ms.live/docs/javascript/v2/plugins/custom-audio-plugins). I was thinking that maybe I can do something in their `processAudioTrack()` method to get the raw binary data. Would this be possible? Any other way on how I can retrieve the binary audio data, or any reading materials you can send would help as well. Thank you!

**Comment by padenot** [comment, marked as answer]
This repository is home to the Web Audio API specification.

Questions on how to do something with the Web Audio API are better suited to either [Stack Overflow](https://stackoverflow.com/questions/tagged/web-audio) or the [Web Audio API slack](https://web-audio-slackin.herokuapp.com/)

**Reply by dmdum** [reply]
Will join the Web Audio API slack, thanks!
